lib/Conf.ml

```diff
@@ -971,6 +972,13 @@ module Formatting = struct
       (fun conf x -> {conf with parse_docstrings= x})
       (fun conf -> conf.parse_docstrings)
 
+  let parse_toplevel_phrases =
+    let doc = "Parse and format toplevel phrases and their output." in
+    let names = ["parse-toplevel-phrases"] in
+    C.flag ~default:false ~names ~doc ~kind
```

Comment:
I think this should be on by default, when `parse-docstrings` is enabled. Especially if we support the case without `;;`.

Reply:
I am not sure we should support the case without `;;`, I am in favor of having this feature opt-in for one or two releases in case we find more bugs, before enabling it by default